pkg/runtime/kubernetes/join_masters.go

```diff
@@ -81,8 +81,8 @@ func (k *Runtime) joinMasters(newMasters []net.IP, master0 net.IP, kubeadmConfig
 			return fmt.Errorf("failed to set join kubeadm config on host(%s) with cmd(%s): %v", m, cmd, err)
 		}
 
-		if err = k.infra.CmdAsync(m, shellcommand.CommandSetHostAlias(k.getAPIServerDomain(), master0.String(), shellcommand.DefaultSealerHostAliasForApiserver)); err != nil {
-			return fmt.Errorf("failed to set hosts alias on(%s): %v", m, err)
+		if err = k.infra.CmdAsync(m, shellcommand.CommandSetHostAlias(k.getAPIServerDomain(), master0.String())); err != nil {
```

Comment:
where we reset master`s own IP for api server ? it will be a bug if we set  master0 ip as api server for each masters.